lib/templates/apollo.config.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+/* eslint-env node */
+// See https://www.apollographql.com/docs/devtools/apollo-config/
+module.exports = {
+  client: {
+    service: {
+      name: 'my-service',
+      // URL to the GraphQL API
+      url: 'http://localhost:3000/graphql',
+    },
+    // Files processed by the extension
+    includes: ['src/**/*.vue', 'src/**/*.js', 'src/**/*.ts'],
+  },
+}
```

Comment:
```suggestion
```
This file looks like a duplicate.

Reply:
The files for JS codebases are generated from TS ones, usually at publish time, and committed.
This generates a duplicate for all files, but without types for JS codebases.
In this case the JS and TS version is the same (being a config file), but I'd have to increase the complexity of the scaffolding code just for one file, I just left it being duplicated